packages/matter.js/src/util/Number.ts

```diff
@@ -42,6 +42,17 @@ export function maxValue<T extends bigint | number>(a: T | undefined, b: T | und
     return a > b ? a : b;
 }
 
-export function toHexString(value: number | bigint) {
+/**
+ * Convert a number or bigint to a hex string which is prefixed by "0x" for logging purposes
+ */
+export function toLogHexString(value: number | bigint) {
```

Comment:
An alternate place to put this could be Diagnostic.hex()

Reply:
Please have a look ... I did it as normal functiopn in the namesace ... you meaned it that way, right?

Reply:
Yep that's right